### PR TITLE
Feature/tickets

### PR DESCRIPTION
**Pull request by SophieTopart**

### CODE REVIEW

File: src/entities/Tickets.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import { Field, ID, ObjectType } from 'type-graphql';
+import User from './Users';
+// import Filelink from './Filelinks';
```
Comment (FredyLy) [inline src/entities/Tickets.ts:3]: Comment to be deleted

File: src/models/Tickets.ts [hunk 2]
```diff
@@ -1,17 +1,16 @@
...
+  id: mongoose.Types.ObjectId,
   subject: String,
   status: String,
-  users: Array,
   deadline: Date,
   description: String,
   initial_time_estimated: Number,
   total_time_spent: Number,
   advancement: Number,
-  file_links: Array
```
Comment (FredyLy) [inline src/models/Tickets.ts:12]: `file_links` is not in `Ticket` model anymore?

File: src/resolvers/TicketsResolver.ts [hunk 3]
```diff
@@ -0,0 +1,74 @@
+/* eslint-disable no-console */
+import { Arg, Query, Resolver, Mutation } from 'type-graphql';
+import Ticket from '../entities/Tickets';
+import TicketsModel from '../models/Tickets';
+import TicketInput from '../inputs/TicketInput';
+import TicketInputUpdate from '../inputs/TicketInputUpdate';
+import IdInput from '../inputs/IdInput';
+// import UsersModel from '../models/Users';
```
Comment (FredyLy) [inline src/resolvers/TicketsResolver.ts:8]: Comment to be deleted

File: src/resolvers/TicketsResolver.ts [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+      return getOneTicket;
+    } catch (err) {
+      return console.log(err);
+    }
+  }
+
+  @Mutation(() => Ticket)
+  async addTicket(@Arg('ticketInput') ticketInput: TicketInput) {
+    try {
+      // const getAllUsers = await UsersModel.find();
```
Comment (FredyLy) [inline src/resolvers/TicketsResolver.ts:35]: Same here

File: src/server.ts [hunk 5]
```diff
@@ -2,14 +2,13 @@ import 'reflect-metadata';
 import { ApolloServer } from 'apollo-server';
 import { buildSchema } from 'type-graphql';
 import UsersResolver from './resolvers/UsersResolver';
+import TicketsResolver from './resolvers/TicketsResolver';
 import ProjectsResolver from './resolvers/ProjectsResolver';
+// import cors from 'cors';
```
Comment (FredyLy) [inline src/server.ts:7]: Delete comment here

**Review by FredyLy — CHANGES_REQUESTED** [review]

File: src/server.ts (hunk 5 above)
Comment (SophieTopart) [reply src/server.ts:7]: OK

File: src/resolvers/TicketsResolver.ts (hunk 4 above)
Comment (SophieTopart) [reply src/resolvers/TicketsResolver.ts:35]: Could have been useful if we have to change the add mutation, but OK

File: src/resolvers/TicketsResolver.ts (hunk 3 above)
Comment (SophieTopart) [reply src/resolvers/TicketsResolver.ts:8]: Could have been useful if we have to change the add mutation, but OK

File: src/entities/Tickets.ts (hunk 1 above)
Comment (SophieTopart) [reply src/entities/Tickets.ts:3]: OK

File: src/models/Tickets.ts (hunk 2 above)
Comment (SophieTopart) [reply src/models/Tickets.ts:12]: Not for now?

**Review by FredyLy — APPROVED** [review]

**Review by grubux — APPROVED** [review]

**Pull request merged**